LayoutTests/http/tests/resourceLoadStatistics/website-data-removal-for-site-navigated-to-with-link-decoration-expected.txt

```diff
@@ -7,7 +7,6 @@ Before deletion: LocalStorage entry does exist.
 Before deletion: IDB entry does exist.
 
 After deletion: HttpOnly cookie exists.
-After deletion: Client-side cookie exists.
```

Comment:
This should instead say "Client-side cookie does not exist." We prefer explicit test output rather than the absence of output meaning something important.